modules/AWS/rds/locals.tf

```diff
@@ -1,5 +1,7 @@
 locals {
-  product_db_name    = var.product_db_name == null ? var.product : var.product_db_name
+  product_db_name   = var.product_db_name == null ? var.product : var.product_db_name
```

Comment:
What is `local.product_db_name` used for? Seems like `var.product_db_name` never get passed in via jira module...

Reply:
I cleaned it up.